src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -0,0 +1,158 @@
+This is called mixed content
+because both HTTP and HTTPS content are being loaded to display the same page,
+and the initial request was secure over HTTPS.
+
+Requesting subresources using the insecure HTTP protocol weakens the security of the entire page,
+as these requests are vulnerable to **man-in-the-middle attacks**,
+where an attacker eavesdrops on a network connection and views or modifies the communication between two parties.
+Using these resources, an attacker can often take complete control over the page,
+not just the compromised resource.
+
+Although many browsers report mixed content warnings to the user,
+by the time this happens, it is too late:
+the insecure requests have already been performed and the security of the page is compromised.
+
+This is why mixed content is increasingly being blocked by browsers.
+If you have mixed content on your site,
+then fixing it will ensure it continues to load as browsers become more strict.
+
+## The two types of mixed content
+
+The two types of mixed content are: active and passive.
+
+**Passive mixed content** refers to content that doesn't interact with the rest of the page,
+and thus a man-in-the-middle attack is restricted to what they can do if they intercept or change that content.
+Passive mixed content includes images, video, and audio content,
+along with other resources that cannot interact with the rest of the page.
+
+**Active mixed content** interacts with the page as a whole and allows an attacker to do almost anything with the page.
+Active mixed content includes scripts, stylesheets, iframes, flash resources,
+and other code that the browser can download and execute.
+
+### Passive mixed content
+
+Passive mixed content is seen as less problematic yet still poses a security threat to your site and your users.
+For example, an attacker can intercept HTTP requests for images on your site and swap or replace these images;
+the attacker can swap the _save_ and _delete_ button images,
+causing your users to delete content without intending to;
+replace your product diagrams with lewd or pornographic content,
+defacing your site; or replace your product pictures with ads for a different site or product.
+
+Even if the attacker doesn't alter the content of your site,
+an attacker can track users using mixed content requests.
+The attacker can tell which pages a user visits and which products they view based on images or other resources that the browser loads.
+
+The following Glitch demo contains examples of passive mixed content.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/passive-mixed-content?path=index.html&previewSize=100"
+    title="passive-mixed-content on Glitch"
+    allow="encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+If passive mixed content is present most browsers will indicate that the page is not secure in the URL bar,
```

Comment:
Maybe 'If passive mixed content is present most browsers will indicate in the URL bar that the page is not secure ...'